### PR TITLE
Ajax js implementation

### PR DESCRIPTION
**Pull request by AdolfoNava**

### CODE REVIEW

**Review by CodeX-y — APPROVED** [review]
good job making a commit after every feature.

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -33,6 +34,7 @@ def create
       if @comment.save
         format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
         format.json { render :show, status: :created, location: @comment }
+        format.js
```
Comment (ahuynh3a) [inline app/controllers/comments_controller.rb:37]: Great job deleting unnecessary white space!

File: app/views/comments/create.js.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+console.log("<%=@comment.body%>")
+// var added_comment = $("<%# j(render @comment) %>");
```
Comment (ahuynh3a) [inline app/views/comments/create.js.erb]: Removed commented-out code that is not necessary for the program to run!

File: app/views/likes/_form.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<%= form_with(model: like, class: "d-inline-block") do |form| %>
+<%= form_with(model: like, class: "d-inline-block", local: false) do |form| %>
```
Comment (ahuynh3a) [inline app/views/likes/_form.html.erb:1]: love that you paid attention to when to use "local:false" and "remote:true"

**Review by ahuynh3a — COMMENTED** [review]
Hi Adolfo! Your code works! Could you go in and remove the unnecessary code and resubmit!

**Review by ahuynh3a — APPROVED** [review]
Thank you!

**Pull request merged**